resources/views/pages/settings/oauth.twig

```diff
@@ -37,12 +37,14 @@
 
                         {{ f.submit(__('form.connect'), {'size' : 'sm', 'icon_left': 'box-arrow-in-right'}) }}
                     </form>
-                {% else %}
+                {% elseif config.allow_user_disconnect %}
                     <form method="post" action="{{ url('/oauth/' ~ name ~ '/disconnect') }}">
                         {{ csrf() }}
 
                         {{ f.submit(__('form.disconnect'), {'btn_type': 'danger', 'size' : 'sm', 'icon_left': 'box-arrow-right'}) }}
                     </form>
+                {% else %}
+                    {{ f.button(__('form.disconnect.disabled'), {'btn_type': 'danger', 'size' : 'sm', 'icon_left': 'box-arrow-right', attr: {'disabled': 'disabled'}}) }}
```

Comment:
I would either not show it at all or just display the text "Disabled" instead. (Definitely preferring the first option)